lang/en.yml

```diff
@@ -141,6 +141,11 @@ en:
     IsNullLabel: 'Is Null'
   SilverStripe\Forms\NumericField:
     VALIDATION: "'{value}' is not a number, only numbers can be accepted for this field"
+  SilverStripe\Forms\SearchableDropdownTrait:
+    SELECT: 'select'
+    TYPE_TO_SEARCH: 'type to search'
+    CONJUNCTIVE: ' or '
+    DOTS: '...'
```

Comment:
Should this be a standard single-character ellipsis?
Does it need to come from a translation file?

Reply:
I think Japanese script uses a different unicode character for the periods in a ellipsis... but I don't know if that makes a case for this being localisable or not? @emteknetnz what was your thinking here? Are you aware of a specific case for this to be localised?

Reply:
Have rewritten to use full sentences rather than joining words together